packages/reusable-blocks/src/store/test/actions.js

```diff
@@ -31,6 +32,7 @@ function createRegistryWithStores() {
 	registry.register( blockEditorStore );
 	registry.register( reusableBlocksStore );
 	registry.register( blocksStore );
+	registry.register( preferencesStore );
```

Comment:
The issue here is that the tests fail because the block editor store uses the preferences store, but it isn't part of the test registry.

Technically `preferences` isn't a dependency of the `reusable-blocks` package, so I'm not sure this is the best way to solve the test issues. I'm surprised the linter doesn't throw an error.

Maybe a mock preferences store would be better.

@adamziel I was wondering if you've encountered this situation before and have any suggestions?

Reply:
@talldan my mental model here is that the `preferences` store actually is a dependency of a `reusable-blocks`, only not a direct one. It is a transitive dependency we get from the `blockEditorStore`. I don't think we have a canonical way of dealing with that when mocking test registries at the moment, but it would be great to have one. 

Thinking out loud, maybe the store object could contain a list of any dependent stores that the tests could iterate over recursively?